jboss-kie-kieserver-webapp-files/added/META-INF/kie-server-jms.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<messaging-deployment xmlns="urn:jboss:messaging-activemq-deployment:1.0">
+    <server name="default">
+        <jms-destinations>
+
+            <!-- custom configuration file for customizable jms configuration -->
+
+            <!-- Kie Server REQUEST queue -->
+            <jms-queue name="##QUEUE_REQUEST_NAME##">
```

Comment:
I hesitate in copying this file just to add replacement strings like "##QUEUE_XXX##", because we'll always have to be checking if the upstream xml code has changed. Can't we re-use and overwrite the existing xml file? For example, instead of replacing values for "##QUEUE_REQUEST_NAME##" and "##QUEUE_REQUEST##, can't we just search and replace "KIE.SERVER.REQUEST", "queue/KIE.SERVER.REQUEST", and "java:jboss/exported/jms/queue/KIE.SERVER.REQUEST" with the right values?

Reply:
I can be done this way, I didn't because doing in this way thinking,  the next steps, enable audit and signal queue/configuration will be easier. Uncomment the respective items on the XMLs will be harder than have something like AUDIT-XXX and replace the config. 
THoughts on how to improve this?

Reply:
You could uncomment the audit and signal config blocks with a regular expression, similar to how I do it here: https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-workbench/added/launch/jboss-kie-workbench.sh#L200

Reply:
OK, let me dig a little on your suggestion and see if it will work well for this case.

Reply:
we will need to have the ejb-jar.xml separated as it contains a few lines to override the jms annotations.

Reply:
Suggested changes applied, @errantepiphany can you please review it again?